### ISSUE TITLE
[0.31-0.32][Android] WebSocket not work with `Debug JS Remotely`

### ISSUE THREAD

**Issue by jhen0409** [facebook/react-native#9523]
Related problem: https://github.com/jhen0409/remote-redux-devtools-on-debugger/issues/14

The WebSocket not work with debug mode on RN v0.31 (v0.32 also) for Android, the following simple code can be reproduced:

```js
const socket = new WebSocket('wss://echo.websocket.org')
socket.onopen = () => socket.send('work fine')
socket.onmessage = evt => alert(evt.data)  // it never call on debug mode
socket.onclose = () => alert('closed')
socket.onerror = () => alert('error')
```

It's work fine without debug mode.

**Comment by ThaJay** [comment]
In my case websockets don't work with or without debug on the current version. Just downgraded my react-native to 29.2 and all is well again.
I chose 29.2 because 30.0 mentions: `Add ping to WebSocket (4ac4f86) - @dbasedow`
https://github.com/facebook/react-native/commit/4ac4f86bf5fa7449434b942d826dd3a13bb9e7d3

**Comment by ThaJay** [comment]
I just upgraded to 32 and websockets seem to be working again for me. I'll unsubscribe from this one.

**Comment by vikeri** [comment]
Can confirm that the problem is solved when upgrading to 0.32.0

**Comment by satya164** [comment]
Should be fixed by this - https://github.com/facebook/react-native/commit/2a5c9a1d897bcc5a9e9565f6f4a3b015453c1cc6

Closing this issue. Let's reopen if you're still facing the issue with 0.32
